### ISSUE TITLE
Issue in 5 solution

### ISSUE THREAD

**Issue by Abhishek404Yadav** [WebDevSimplified/Learn-SQL#18]
/*SOLUTION 5*/
SELECT bands.name AS "band name"
FROM bands 
LEFT JOIN albums
ON bands.id=albums.band_id
GROUP BY bands.name 
HAVING COUNT(albums.id)=0;
/*including bands.name in group by is important as it is in result*/

**Comment by ReneGombar** [comment]
Yes, that works because in the exercise band names are unique. In a different scenario where you have multiple bands with the same name, the GROUP BY should be using the band's unique IDs ( GROUP BY bands.id )
